app/views/pages/index.html.erb

```diff
@@ -16,7 +16,6 @@
   </li>
 </ul>
 <hr/>
-<div id="content"></div>
-
-
-
+<div id="content">
```

Comment:
Actually, we don't need it. But we haven't looked at if we need it for any styling.